packages/terra-consumer-nav/src/NavPropShapes.js

```diff
@@ -6,14 +6,18 @@ const navItemShape = {
    */
   text: PropTypes.string.isRequired,
   /**
-   * The path the profile link would redirect to.
+   * The path the link would redirect to.
    */
-  uri: PropTypes.string.isRequired,
+  url: PropTypes.string,
```

Comment:
ah you are correct here, we were previously bypassing the required field by simply passing `url: '',` into the component. 
example: https://github.com/cerner/terra-consumer/blob/master/packages/terra-consumer-nav/tests/nightwatch/DefaultNav.jsx#L37